### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by gdalle**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -3,5 +3,5 @@ Franklin = "713c75ef-9fc9-4b05-94a9-213340da978e"
 NodeJS = "2bd173c7-0d6d-553b-b6af-13a54713934c"
 
 [compat]
-Franklin = "0.10.28"
-NodeJS = "1.1.1"
+Franklin = "0.10"
+NodeJS = "1.1"
```
Comment (oxinabox) [inline Project.toml:7]: Why this change?

**Review by oxinabox — APPROVED** [review]

File: Project.toml (hunk 1 above)
Comment (gdalle) [reply Project.toml:7]: old habit, I usually put minor versions in compat fields, but I can revert
